services/cirrus/cirrus.service.js

```diff
@@ -0,0 +1,76 @@
+'use strict'
+
+const Joi = require('joi')
+const { isBuildStatus, renderBuildStatusBadge } = require('../build-status')
+const { BaseJsonService } = require('..')
+
+const schema = Joi.object({
+  message: Joi.alternatives()
+    .try(isBuildStatus, Joi.equal('unknown'))
+    .required(),
+}).required()
+
+module.exports = class Cirrus extends BaseJsonService {
+  static get category() {
+    return 'build'
+  }
+
+  static get route() {
+    return {
+      base: 'cirrus',
+      pattern: '/github/:user/:repo',
+    }
+  }
+
+  static get examples() {
+    return [
+      {
+        title: 'Cirrus CI - Base Branch Build Status',
+        namedParams: { user: 'flutter', repo: 'flutter' },
+        staticPreview: this.render({ subject: 'Ci', status: 'passing' }),
+      },
+      {
+        title: 'Cirrus CI - Specific Branch Build Status',
+        pattern: '/github/:user/:repo?branch=:branch',
+        namedParams: { user: 'flutter', repo: 'flutter', branch: 'master' },
+        staticPreview: this.render({ subject: 'Ci', status: 'passing' }),
+      },
+      {
+        title: 'Cirrus CI - Specific Task Build Status',
+        pattern: '/github/:user/:repo?task=:task',
+        namedParams: { user: 'flutter', repo: 'flutter', task: 'analyze' },
+        staticPreview: this.render({ subject: 'analyze', status: 'passing' }),
+      },
+      {
+        title: 'Cirrus CI - Task and Script Build Status',
+        pattern: '/github/:user/:repo?task=:task&script=:script',
+        namedParams: {
+          user: 'flutter',
+          repo: 'flutter',
+          task: 'analyze',
+          script: 'test',
+        },
```

Comment:
Can't get the multi-line suggestion replacement to work, but you'll want to change this object to drop the query params

```js
        namedParams: { user: 'flutter', repo: 'flutter' },
```